hummingbird/ml/_container.py

```diff
@@ -275,3 +281,138 @@ def score_samples(self, *inputs):
         f = super(TorchScriptSklearnContainerAnomalyDetection, self).score_samples
 
         return _torchscript_wrapper(device, f, *inputs)
+
+
+# ONNX containers.
+class ONNXSklearnContainer(ABC):
+    """
+    Base container for ONNX models.
+    The container allows to mirror the Sklearn API.
+    """
+
+    def __init__(self, model, extra_config={}):
+        """
+        Args:
+            model: A ONNX model
+            extra_config: Some additional configuration parameter
+        """
+        if onnx_runtime_installed():
```

Comment:
i think we need another fail/`else` case here, else it will die on like 305 if this is false

Reply:
Yes I definitely forgot this one.